### ISSUE TITLE
NixOS formula please?

### ISSUE THREAD

**Issue by pshirshov** [ttalvitie/browservice#87]
That would be very nice to have.

**Comment by ttalvitie** [comment]
As I don't have any experience or links with NixOS, I won't provide NixOS packaging for Browservice, but I can help if some issues come up when packaging. Does the AppImage work on NixOS?

**Comment by pshirshov** [comment]
> Does the AppImage work on NixOS?

More no than yes. NixOS is a very special OS with completely different approach to library linking. While there are ways to run appimages there, they are non-native and fragile.

**Comment by ttalvitie** [comment]
Right, should have known as much. I would still be interested in hearing whether the AppImage works on NixOS using any of the available shims; the Browservice AppImage attempts to be a bit more self-contained than most AppImages (because it encapsulates a graphical browser program but needs to be able to run in non-graphical installations).

If someone wants to create a proper Nix package for Browservice, one good starting point would be to see whether some other software based on CEF (Chromium Embedded Framework) are packaged for Nix, because it's probably the most complicated and time-consuming part of Browservice to build (Browservice uses a patched version of it).
